distribution/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/tests/integration/ServerAdminTestCase.java

```diff
@@ -95,12 +97,20 @@ public void testServerStateChangeErrorScenario() throws Exception {
         log.debug("Logged-in cookie : " + sessionCookie);
         String url = UrlGenerationUtil.getLoginURL(automationContext.getDefaultInstance()) +
                 "server-admin/proxy_ajaxprocessor.jsp?action=shutdown";
-        URL obj = new URL(url);
-        HttpURLConnection con = (HttpURLConnection) obj.openConnection();
-        con.setRequestMethod("GET");
-        con.setRequestProperty("Cookie", sessionCookie);
-        int responseCode = con.getResponseCode();
-        assertEquals(responseCode, HttpStatus.SC_METHOD_NOT_ALLOWED);
+
+        HttpClient httpClient = HttpClient.newBuilder()
+                .version(HttpClient.Version.HTTP_1_1)
+                .build();
+
+        HttpRequest request = HttpRequest.newBuilder()
+                .GET()
+                .uri(URI.create(url))
+                .setHeader("Cookie", sessionCookie) // add request header
```

Comment:
Format the comment

Reply:
Fixed by - https://github.com/wso2/carbon-kernel/pull/3571/commits/3d285191435cf948c34262bfbc1097aceaa78339
Removed that comment as that comment is not necessary